.travis.yml

```diff
@@ -5,6 +5,7 @@ php:
     - 5.6
     - 5.5
     - 5.4
+    - 5.3.3
```

Comment:
you should test on 5.3 rather than 5.3.3. The 5.3.3 image on Travis does nont have OpenSSL